src/status_im/contexts/profile/events.cljs

```diff
@@ -27,7 +28,10 @@
 (rf/reg-fx
  :profile/get-profiles-overview
  (fn [callback]
-   (native-module/open-accounts callback)))
+   (native-module/initialize-application {:dataDir       (native-module/backup-disabled-data-dir)
+                                          :mixpanelAppId config/mixpanel-app-id
```

Comment:
I don't think we will abstract credentials of providers eventually, I have only added 2 on status-go because i started with appsflyer, but got stuck on a support request and it was faster to move to mixpanel then waiting for them to reply, but I would remove it eventually (waiting to hear if anyone is really keen on them, but seems like we are ok) if we are not using it.
AppID/Secret I am also not 100% sure it's a valid abstraction, it makes sense with the two providers I have tried,  but I am not sure it applies for every provider, is it ok if we keep it for now and then once the dust settles revisit?

Reply:
I just thought about a code like below, meaning the config is responsible for setting up the var since it's where the values actually are (like token and app id), then the vars can even be kept private to reinforce isolation.

```clojure
(native-module/initialize-application
 (merge {:dataDir (native-module/backup-disabled-data-dir)}
        config/centralized-metrics-settings)
 callback)
```

But it's good as is Andrea, I don't want to overcomplicate too :)